bingo/bingo-elastic/python/bingo_elastic/elastic.py

```diff
@@ -151,13 +154,12 @@ def prepare(
 
 
 def response_to_records(
-    res, index_name, postprocess_actions
+    res, index_name, postprocess_actions, indigo_session=None, options=""
```

Comment:
Could you please add the type hints here?

Reply:
My bad, already added.